app/src/main/AndroidManifest.xml

```diff
@@ -26,6 +26,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar"
       android:screenOrientation="portrait"
       android:windowSoftInputMode="adjustResize"/>
+    <activity
+      android:name=".testing.ExplorationTestActivity" />
```

Comment:
Please maintain alphabetical order for activities.

Reply:
Done.